lib/smtpapi.rb

```diff
@@ -116,20 +116,20 @@ def set_ip_pool(pool_name)
 
     def to_array
       data = {}
-      data['to'] = @to if @to.length > 0
```

Comment:
It looks like the Travis build is having some issues with the .emtpy? calls here: https://travis-ci.org/sendgrid/smtpapi-ruby/builds/294550416#L652

Can you please look into this and see what's causing these errors?